tests/zfs-tests/tests/functional/compression/compress_003_pos.ksh

```diff
@@ -62,7 +62,7 @@ typeset -i offset=0
 
 for propname in "compression" "compress"
 do
-	for value in $(get_compress_opts zfs_compress)
+	for value in "${compress_prop_vals[@]:1}"
```

Comment:
Is this syntax valid ksh?  I was under the impression `${arr[@]:s:n}` was a bash'ism.  Though I'd be happy to be wrong about that.

Reply:
This actually took me some tries...
If “off” (first in that array) is included it errors out, so I actually had to check to make sure it actually skipped the first value. :)

Reply:
I had the same thought, and verified that it works on ksh.

Reply:
Its nice to know, gives a lot of options!